src/components/Chart/BarChart.js

```diff
@@ -0,0 +1,19 @@
+import { AreaChart } from './index';
+import { CHART_CONFIG } from './constants';
+
+/**
+ * BarChart Component for Patternfly React
+ */
+class BarChart extends AreaChart {
```

Comment:
BarChart extends AreaChart... hmm, I don't know? Maybe just call the base class Chart and have that extend C3Chart? 

Are we going classes instead of stateless here now?

Reply:
Apologies - thanks for your patience @yaacov.

Reply:
> Maybe just call the base class Chart

👍 using `BaseChart` now in the base class

Reply:
> classes instead of stateless here

@priley86 
Good point, stateless should be first choice. But in out case, c3chart is already a class [1] so in this case we lose the optimisations we can get from using stateless, even if we use a stateless function. On the other hand we do need the life-cycle methods to control the data and props changes.

[1] https://github.com/bcbcarl/react-c3js/blob/master/src/index.js